boot/boot.go

```diff
@@ -163,16 +163,21 @@ func applicable(s snap.PlaceInfo, t snap.Type, dev Device) bool {
 	return true
 }
 
-// bootState exposes the boot state for a type of boot snap.
+// bootState exposes the boot state for a type of boot snap during
+// normal running state, i.e. after the pivot_root and after the initramfs.
 type bootState interface {
 	// revisions retrieves the revisions of the current snap and
 	// the try snap (only the latter might not be set), and
 	// the status of the trying snap.
+	// Note that the error could be ErrorKindTrySnap, in which case curSnap may
```

Comment:
ErrorKindTrySnap doesn't exist anymore, this need to be reformulated

Reply:
yes I missed this thanks for spotting